settings.gradle.kts

```diff
@@ -80,14 +69,30 @@ gradleEnterprise {
     }
 }
 
+gitRepositories {
+    checkoutsDirectory.set(file(rootProject.projectDir.absolutePath + "/hedera-node/hapi/"))
+    include("hedera-protobufs") {
+        uri.set("https://github.com/hashgraph/hedera-protobufs.git")
+        // choose tag or branch of HAPI you would like to test with
+        // This version needs to match tha HAPI version below in versionCatalogs
+        tag.set("v0.34.0")
+        // do not load project from repo
+        autoInclude.set(false)
+    }
+}
+
 // Define the library catalogs available for projects to make use of
 dependencyResolutionManagement {
     @Suppress("UnstableApiUsage")
     versionCatalogs {
         // The libs of this catalog are the **ONLY** ones that are authorized to be part of the runtime
         // distribution. These libs can be depended on during compilation, or bundled as part of runtime.
         create("libs") {
+            // The HAPI API version to use, this need to match the tag set on gitRepositories above
+            version("hapi-version", "0.34.0")
```

Comment:
it would be great if `0.34.0`could be externalized as a property that is here and for `tag.set("v0.34.0")` (would be `tag.set("v" + hedera-protobuf-api-version)`)

Reply:
I really tried but could not find a way that gradle was happy with